modules/about-workbench-images.adoc

```diff
@@ -38,44 +38,39 @@ endif::[]
 | Image name | Description
 
 | CUDA
-| If you are working with compute-intensive data science models that require GPU support, use the Compute Unified Device Architecture (CUDA) notebook image to gain access to the NVIDIA CUDA Toolkit. Using this toolkit, you can optimize your work by using GPU-accelerated libraries and optimization tools.
+| If you are working with compute-intensive data science models that require GPU support, use the Compute Unified Device Architecture (CUDA) workbench image to gain access to the NVIDIA CUDA Toolkit. Using this toolkit, you can optimize your work by using GPU-accelerated libraries and optimization tools.
 
 | Standard Data Science
-| Use the Standard Data Science notebook image for models that do not require TensorFlow or PyTorch. This image contains commonly-used libraries to assist you in developing your machine learning models.
+| Use the Standard Data Science workbench image for models that do not require TensorFlow or PyTorch. This image contains commonly-used libraries to assist you in developing your machine learning models.
 
 | TensorFlow
 | TensorFlow is an open source platform for machine learning. With TensorFlow, you can build, train and deploy your machine learning models. TensorFlow contains advanced data visualization features, such as computational graph visualizations. It also allows you to easily monitor and track the progress of your models.
 
 | PyTorch
-| PyTorch is an open source machine learning library optimized for deep learning. If you are working with computer vision or natural language processing models, use the Pytorch notebook image.
+| PyTorch is an open source machine learning library optimized for deep learning. If you are working with computer vision or natural language processing models, use the Pytorch workbench image.
 
 | Minimal Python
 | If you do not require advanced machine learning features, or additional resources for compute-intensive data science work, you can use the Minimal Python image to develop your models.
 
 | TrustyAI
-| Use the TrustyAI notebook image to leverage your data science work with model explainability, tracing, and accountability, and runtime monitoring. See the link:https://github.com/trustyai-explainability/trustyai-explainability-python-examples[TrustyAI Explainability repository^] for some example Jupyter notebooks.
+| Use the TrustyAI workbench image to leverage your data science work with model explainability, tracing, and accountability, and runtime monitoring. See the link:https://github.com/trustyai-explainability/trustyai-explainability-python-examples[TrustyAI Explainability repository^] for some example workbenches.
```

Comment:
```suggestion
| Use the TrustyAI workbench image to leverage your data science work with model explainability, tracing, and accountability, and runtime monitoring. See the link:https://github.com/trustyai-explainability/trustyai-explainability-python-examples[TrustyAI Explainability repository^] for some example Jupyter notebooks.
```

opening that link, you'd really find example jupyter notebooks (.ipynb files) there